subsys/bluetooth/host/Kconfig

```diff
@@ -47,6 +47,13 @@ config BT_RX_BUF_LEN
 	  an L2CAP MTU of 65 bytes. On top of this there's the L2CAP header
 	  (4 bytes) and the ACL header (also 4 bytes) which yields 73 bytes.
 
+config BT_BUF_ACL_DATA_LEN
+	int
+	range 27 251
+	default 65
+	depends on BT_HCI_RAW
+	prompt "Override the maximum ACL data payload." if BT_HCI_RAW
```

Comment:
I don't think the if BT_HCI_RAW is needed here when you have depends on BT_HCI_RAW.